frontend/src/components/map/ZaehlstelleMap.vue

```diff
@@ -213,11 +213,25 @@ export default class ZaehlstelleMap extends Vue {
         return this.$store.getters["search/result"];
     }
 
+    get zoomValue() {
+        const zoom = this.$router.currentRoute.query.zoom;
+
+        if (zoom != undefined) {
+            return parseFloat(zoom.toString());
+        } else {
+            return this.zoom;
```

Comment:
Da oben die Prüfung `zoom != undefined` ist, bedeutet dass, dass du hier undefined zurück gibst. Hier wäre es besser einen default zurückzugeben

Reply:
--> commit [9b4f7ab](https://github.com/it-at-m/dave-frontend/pull/47/commits/9b4f7ab1d2800799d525082e02692c0558be6ee5)

Reply:
zoom und this.zoom sind nicht das gleiche, daher in commit [7bb673c](https://github.com/it-at-m/dave-frontend/pull/47/commits/7bb673c2a91df3d54006a41ae019425d60ea5554) wieder reverted